react-example/src/index.tsx

```diff
@@ -38,8 +38,16 @@ const HomeLink = styled(Link)`
 `;
 
 ((): void => {
+  config.setConfig({
+    isEuIterableService: true,
+    dangerouslyAllowJsPopups: true
+  });
   const { setEmail, logout, refreshJwtToken } = initialize(
     process.env.API_KEY || '',
+    {
+      isEuIterableService: false,
+      dangerouslyAllowJsPopups: true
+    },
```

Comment:
What's the usage of these two sets of settings? Can you just pass `config` into the initialize fn after you've set it? But looks like they differ in the EU value, is that intentional?

Reply:
oops, meant to remove that.